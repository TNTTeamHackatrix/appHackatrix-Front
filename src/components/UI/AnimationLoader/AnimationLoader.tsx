import React from "react";
import styles from "./AnimationLoader.module.scss";

class AnimationLoader extends React.Component {
  render() {
    return (
      <div className={styles.Lds}>
        <div className={styles.LdsSpinner}>
          <div/>
          <div/>
          <div/>
          <div/>
          <div/>
          <div/>
          <div/>
          <div/>
          <div/>
          <div/>
          <div/>
          <div/>
        </div>
      </div>
    )
  }
}

export default AnimationLoader;
