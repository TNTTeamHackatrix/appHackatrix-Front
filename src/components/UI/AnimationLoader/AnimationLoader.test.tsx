import React from "react";
import renderer from "react-test-renderer";
import AnimationLoader from "./AnimationLoader";

describe("AnimationLoader", () => {

  it("always renders a div tag", () => {
    const AnimationLoaderComponent = renderer.create(<AnimationLoader/>).toJSON();
    expect(AnimationLoaderComponent).toMatchSnapshot();
  });

});