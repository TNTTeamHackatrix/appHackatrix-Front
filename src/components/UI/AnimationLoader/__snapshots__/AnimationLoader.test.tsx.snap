// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnimationLoader always renders a div tag 1`] = `
<div
  className="Lds"
>
  <div
    className="LdsSpinner"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
